src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,80 @@
+package duke.command;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.exception.FindException;
+import duke.task.Task;
+
+public class FindCommand extends Command {
+
+    private String[] keywords;
+
+    public FindCommand(String... keywords) {
+        this.keywords = keywords;
+    }
+
+    @Override
+    public String execute(TaskList taskList, Ui ui, Storage storage) {
+        try {
+            return processFind(this.keywords, taskList, ui, storage);
+
+        } catch (FindException find) {
+            return find.getMessage();
+        }
+    }
+
+    public String processFind(String[] toFind, TaskList taskList, Ui ui, Storage storage) throws FindException {
```

Comment:
Perhaps the String[] toFind could be better named to reflect the variable it contains?